lib/assets/javascripts/twine.coffee

```diff
@@ -69,13 +70,29 @@
       element = findOrCreateElementForNode(node)
       element.indexes = indexes
 
-    for type, binding of Twine.bindingTypes when definition = Twine.getAttribute(node, type)
-      element = findOrCreateElementForNode(node)
+    bindingConstructors = null
```

Comment:
This one is needed since we want to use a `?=` inside the for loop.

```
error: the variable "bindingConstructors" can't be assigned with ?= because it has not been declared before
                  bindingConstructors ?= []
```